src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -206,15 +210,16 @@ def __call__(
             - **pixel_values** -- Pixel values to be fed to a model.
             - **pixel_mask** -- Pixel mask to be fed to a model (when `pad_and_return_pixel_mask=True` or if
               *"pixel_mask"* is in `self.model_input_names`).
-            - **mask_labels** -- Optional mask labels of shape `(batch_size, num_classes, height, width) to be fed to a
-              model (when `annotations` are provided).
-            - **class_labels** -- Optional class labels of shape `(batch_size, num_classes) to be fed to a model (when
-              `annotations` are provided).
+            - **mask_labels** -- Optional a list of mask labels of shape `(labels, height, width)` to be fed to a model
```

Comment:
```suggestion
            - **mask_labels** -- Optional list of mask labels of shape `(labels, height, width)` to be fed to a model
```

Reply:
What do you mean by (labels, ...) here => number of labels?

Reply:
yes, the number of labels present in that mask